src/context/build-services.js

```diff
@@ -1,29 +1,34 @@
-    .addInstance('forestServerRequester', () => require('../services/forest-server-requester'))
+    .addInstance('ipWhitelist', () => require('../services/ip-whitelist')) // Could be handle by ForestAdminClient
+    .addInstance('forestServerRequester', () => require('../services/forest-server-requester')) // Could be handle by ForestAdminClient
     .addInstance('schemasGenerator', () => require('../generators/schemas'))
     .addInstance('baseFilterParser', () => require('../services/base-filters-parser'))
-    .addInstance('permissionsFormatter', () => require('../services/permissions-formatter'))
     .addUsingClass('projectDirectoryFinder', () => require('../services/project-directory-finder'))
     .addUsingClass('configStore', () => require('../services/config-store'))
-    .addUsingClass('permissionsGetter', () => require('../services/permissions-getter'))
-    .addUsingClass('permissionsChecker', () => require('../services/permissions-checker'))
     .addUsingClass('apimapFieldsFormater', () => require('../services/apimap-fields-formater'))
-    .addUsingClass('authorizationFinder', () => require('../services/authorization-finder'))
-    .addUsingClass('apimapSorter', () => require('../services/apimap-sorter'))
-    .addUsingClass('apimapSender', () => require('../services/apimap-sender'))
+    .addUsingClass('authorizationFinder', () => require('../services/authorization-finder')) // Could be handle by ForestAdminClient
+    .addUsingClass('apimapSorter', () => require('../services/apimap-sorter')) // Could be handle by ForestAdminClient
+    .addUsingClass('apimapSender', () => require('../services/apimap-sender')) // Could be handle by ForestAdminClient
     .addUsingClass('schemaFileUpdater', () => require('../services/schema-file-updater'))
-    .addUsingClass('scopeManager', () => require('../services/scope-manager'))
+    .addUsingClass('scopeManager', () => require('../services/scope-manager')) // Will be handle by ForestAdminClient
     .addUsingClass('modelsManager', () => require('../services/models-manager'))
-    .addUsingClass('tokenService', () => require('../services/token'))
-    .addUsingClass('oidcConfigurationRetrieverService', () => require('../services/oidc-configuration-retriever'))
-    .addUsingClass('oidcClientManagerService', () => require('../services/oidc-client-manager'))
-    .addUsingClass('authenticationService', () => require('../services/authentication'))
+    .addUsingClass('tokenService', () => require('../services/token')) // Could be handle by ForestAdminClient
+    .addUsingClass('oidcConfigurationRetrieverService', () => require('../services/oidc-configuration-retriever')) // Could be handle by ForestAdminClient
+    .addUsingClass('oidcClientManagerService', () => require('../services/oidc-client-manager')) // Could be handle by ForestAdminClient
+    .addUsingClass('authenticationService', () => require('../services/authentication')) // Could be handle by ForestAdminClient
```

Comment:
Should we remove those comments for now?

Reply:
Yes, I think we should